src/dbt/kipptaf/models/extracts/tableau/rpt_tableau__dibels_dashboard.sql

```diff
@@ -506,4 +506,456 @@ select
+            ,s.ethnicity
+            ,s.Homeless
+            ,s.is_504
+            ,s.sped
+            ,s.lep
+            ,s.lunch_status
+            ,m.teacher_id
+            ,m.teacher_name
+            ,m.course_name
+            ,m.course_number
+            ,m.section_number
+            ,m.expected_test
+            ,case when m.student_number_schedule is null then 0
+                  else 1 end as enrolled_but_not_scheduled --tagging students as enrolled in school but not scheduled for the course that would allow them to take the diBels test
+            ,case when m.student_number_schedule is not null and d.mclass_student_number is null then 0
+                  else 1 end as student_has_assessment_data --tagging students who are schedule for the correct class but have not tested yet for the expected assessment term
+            ,d.mclass_district_name
+            ,d.mclass_school_name
+            ,d.mclass_student_number
+            ,case when d.mclass_period in ('BOY','MOY','EOY') then d.mclass_student_number
+                  else null end as mclass_student_number_Bm --another tag for students who tested for benchmarks
+            ,case when d.mclass_period in ('BOY->MOY','MOY->EOY') then d.mclass_student_number
+                  else null end as mclass_student_number_pm --another tag for students who tested for progress monitoring
+            ,d.mclass_student_last_name
+            ,d.mclass_student_first_name
+            ,d.mclass_enrollment_grade
+            ,d.mclass_reporting_class_name
+            ,d.mclass_reporting_class_id
+            ,d.mclass_official_teacher_name
+            ,d.mclass_official_teacher_staff_id
+            ,d.mclass_assessing_teacher_name
+            ,d.mclass_assessing_teacher_staff_id
+            ,d.mclass_assessment
+            ,d.mclass_assessment_edition
+            ,d.mclass_assessment_grade
+            ,d.mclass_period
+            ,d.mclass_client_date
+            ,d.mclass_sync_date
+            ,d.mclass_probe_number
+            ,d.mclass_total_number_of_probes
+            ,d.mclass_measure
+            ,cast(d.mclass_measure_score as string) as mclass_measure_score
+            ,d.mclass_score_change
+            ,d.mclass_measure_level
+            ,d.measure_percentile
+            ,d.measure_semester_growth
+            ,d.measure_year_growth
+      from student as s 
+      left join schedule as m
+      on s.enrollment_academic_year = m.schedule_academic_year and s.student_number_enrollment = m.student_number_schedule
+      left join assessments_scores as d
+      on m.schedule_academic_year = d.mclass_academic_year and m.student_number_schedule = d.mclass_student_number and m.expected_test = d.mclass_period --this last join on field is to ensure rows are generated for all expected tests, even if the student did not assess for them
+) as b
+where b.enrollment_grade_level not in ('3','4') --excluding 3rd and 4th graders until iready scores are available
```

Comment:
no need for this really if you plan to add iready data soon

Reply:
I dont have a roster for 3rd/3th graders yet because they are pending starting BOY iReady.  This is my way to keep that in mind but away from the draft dashboard. Is that not acceptable?

Reply:
It's not breaking anything, but I'd recommend filtering them on the dashboard side instead of having to do a code change later on when iReady data is coming in